src/frontend/packages/core/src/shared/components/list/list-types/endpoint/base-endpoints-data-source.ts

```diff
@@ -31,13 +31,13 @@ export function syncPaginationSection(
 
 export class BaseEndpointsDataSource extends ListDataSource<EndpointModel> {
   store: Store<CFAppState>;
-  endpointType: string;
+  cnsiType: string;
```

Comment:
Why did you rename this to cnsi? I thought we were trying to move away from that term?

Reply:
There's a conflict with the new endpointType property added to the base ListDataSource. Renamed it here to cnsi as we're comparing later to a cnsi_type.